spec/requests/stopwatches_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'Stopwatches', type: :request do
+  let!(:user) { create(:user) }
+
+  describe 'GET #index' do
+    context 'when user is logged in' do
+      before { sign_in(user) }
+
+      it 'returns a success response' do
+        get stopwatches_path
+        expect(response).to be_successful
+      end
+
+      context 'has stopwatch' do
+        let!(:stopwatch) { create(:stopwatch, user: user) }
+
+        it 'shows stopwatch saved' do
+          get stopwatches_path
+          expect(response.body).to include(stopwatch.name)
+        end
+      end
+    end
+
+    context 'when user is not signed in' do
+      before { get stopwatches_path }
+
+      it 'does not render a page' do
+        expect(response).not_to be_successful
+      end
+
+      it 'is redirected to sign in' do
+        expect(response).to redirect_to(new_user_session_path)
+      end
+    end
+  end
+
+  describe 'POST #create' do
+    let(:stopwatch_valid_params) do
+      {
+        time: '2021/17/03 13:00:00',
+        name: 'Stopwatch test',
+        laps: ['00:30:00', '00:25:00'],
+        user_id: user.id
+      }
+    end
```

Comment:
Avoid hardcoded values; you can generate these values randomly, defining a pattern for them.